src/sequences/longest-common-subsequence.md

```diff
@@ -0,0 +1,203 @@
+# Longest Common Subsequence
```

Comment:
Please use proper [tags](https://cp-algorithms.com/contrib.html#tags) for the file.

Reply:
done